# Flight Control
float32 rollRate
float32 yawRate
float32 pitchRate
float32 z

